scalafix/input/src/main/scala/fix/Fs2sinkremoval.scala

```diff
@@ -0,0 +1,16 @@
+/*
+rule = v1
+*/
+package fix
+
+import fs2._
+import fs2.{Pipe, Sink, Stream}
+import fs2.{Sink => X, Stream}
```

Comment:
Did you intend to add a `X[F, A]` example below?

Reply:
Not sure about this one.  @monadplus what was the intent of the `Sink` rename?